addon/routes/dashboard/index.js

```diff
@@ -0,0 +1,10 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class DashboardIndexRoute extends Route {
```

Comment:
I'm not sure this route is needed. Since `dashboard` redirects I don't think this will ever get called.